src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -134,6 +122,35 @@ override void OnLaunched(LaunchActivatedEventArgs e)
 			DisplayLaunchArguments(e);
 		}
 
+		private static void ProcessEventArgs(LaunchActivatedEventArgs e)
+		{
+#if __SKIA__
+			var runAutoScreenshotsParam =
+			e.Arguments.Split(';').FirstOrDefault(a => a.StartsWith("--auto-screenshots"));
+
+			var screenshotsPath = runAutoScreenshotsParam?.Split('=').LastOrDefault();
+#endif
+
+			var sw = Stopwatch.StartNew();
```

Comment:
Codacy found an issue: [Remove the unused local variable 'sw'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932478)